src/XamlStyler/Options/StylerOptions.cs

```diff
@@ -356,6 +356,29 @@ public string ConfigPath
         [Browsable(false)]
         public bool SuppressProcessing { get; set; }
 
+        private string endOfLine;
+
+        [Category("Misc")]
+        [RefreshProperties(RefreshProperties.All)]
+        [Description("Defines end of line character. Specify 'lf' or 'crlf'; otherwise, default character of the host will be used.")]
+        [JsonProperty("EndOfLine", DefaultValueHandling = DefaultValueHandling.IgnoreAndPopulate)]
+        [DefaultValue("")]
+        public string EndOfLine
```

Comment:
Instead of exposing this setting in the UI at all, I am wondering if this setting should follow the pattern of UseVisualStudioIndentSize/IndentSize (i.e., do not expose in settings UI, but still allow IDE to be overwritten in config file)?

Though, in this case, we wouldn't need ```UseVisualStudioLineEndings``` because we don't need to look it up from the XAML editor properties. Instead, if ```LineEndings``` is unset in a configuration file, ```NewLine``` is set to ```Environment.NewLine``` (current behavior), but then it could be overwritten. Instead of accepting a string, follow pattern of options like ```AttributeIndentationStyle``` and create an enum with: CRLF, LF, CR.